README.md

```diff
@@ -29,89 +17,107 @@ Thank you to [Radek Tomšej](https://github.com/tomsej) for initial development
  </a>
 </p>
 
-List of [pre-commit](https://pre-commit.com) hooks to ensure the quality of your [dbt](https://www.getdbt.com) projects.
+## Sponsors
+
+<img src=".github/datacoves-logo.png" alt="Datacoves" width="150"/><br>
+The turnkey analytics stack, find out more at [datacoves.com](https://datacoves.com/).
+
+<br>
+
+<img style="margin-top: 20px;" src=".github/montreal-analytics-logo.png" alt="Montreal Analytics" width="150"/><br>
+[Montreal Analytics](https://www.montrealanalytics.com/?utm_source=blog&utm_medium=medium&utm_campaign=dbt-checkpoint) is a full-stack data consultancy servicing North America, and are both a dbt Preferred Consulting Partner and Platinum dbt Certification Award winner.
 
 ## Goal
 
-*Quick ensure the quality of your `dbt` projects*.
+[pre-commit](https://pre-commit.com) hooks to ensure the quality of your [dbt](https://www.getdbt.com) projects.
 
 `dbt` is awesome, but when a number of models, sources, and macros grow it starts to be challenging to maintain quality. People often forget to update columns in schema files, add descriptions, or test. Besides, with the growing number of objects, dbt slows down, users stop running models/tests (because they want to deploy the feature quickly), and the demands on reviews increase.
```

Comment:
`dbt` is awesome, but when the number of models, sources, and macros in a project grows, it becomes challenging to maintain the same level of quality across developers.. Users forget to update columns in property(yml) files or add table and column add descriptions. Without automation the reviewer workload increases and unintentional errors may be missed. dbt-checkpoint allows organizations to add automated validations improving your code review and release process.